### DISCUSSION TITLE
How to set Grid row height globally?

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6637] [Help] [answered]
For a single grid, we can change row height using getSlickOptions RowHeight property. Is there a way to apply this change across ALL grids in my solution without having to do each one individually?

**Comment by wezmag** [comment, marked as answer]
Add below in `\Modules\Common\ScriptInit.ts` . 
``` typescript
DataGrid.defaultRowHeight = 100;
```
[source code here](https://github.com/serenity-is/Serenity/blob/5c5919f8955bee406487b68fff16ec7e52509152/src/Serenity.Scripts/corelib/src/ui/datagrid/datagrid.ts#L112)
